### ISSUE TITLE
Give buttons roles

### ISSUE THREAD

**Issue by ben9583** [harshavb/piously#32]
The next step for the game is to define what each button will do. We should map out a goal for every button when the user clicks it before we start work on the implementation

Issues for each button:
Exit: #34

**Comment by harshavb** [comment]
Likely singleplayer, casual, ranked, settings, leaderboards, and exit, of which we implement in the order exit -> settings -> singleplayer -> casual -> leaderboards -> ranked

**Comment by ben9583** [comment]
Leaderboards cuts casual and ranked? Maybe having single/casual/ranked on the top half, settings on bottom left, leaderboard on bottom right, and exit on bottom

**Comment by harshavb** [comment]
No that's the order of implementation, not the order of appearance

**Comment by ben9583** [comment]
Oh I see, yea that's a good idea then. I'll create a new issue for the exit button functionality.

**Comment by ben9583** [comment]
Just thought about it some more; @harshavb how about instead of doing casual/ranked, we do local/online multiplayer, with the online multiplayer having those casual/ranked options?

**Comment by harshavb** [comment]
Good idea

**Comment by ben9583** [comment]
Decided upon; closing issue
